app/src/main/java/com/codingblocks/cbonlineapp/course/CourseActivity.kt

```diff
@@ -193,6 +206,23 @@ class CourseActivity : BaseCBActivity(), AnkoLogger, AppBarLayout.OnOffsetChange
             courseCardListAdapter.submitList(courses)
         }
 
+        viewModel.wishlistUpdated.observer(this) {
+            if (getPrefs().SP_JWT_TOKEN_KEY.isNotEmpty()) {
```

Comment:
this check is not required as this will be only triggered if the user is logged in